src/EnergyPlus/PhotovoltaicThermalCollectors.hh

```diff
@@ -77,6 +77,8 @@ namespace PhotovoltaicThermalCollectors {
         FIXED
     };
 
+    extern int NumPVT; // number of transpired collectors in model
```

Comment:
Not sure why I am only getting to this PR now but ... here goes. No free-floating global variables like this. This should be part of `state`.

Reply:
updated

Reply:
Did your change get pushed?  It's still showing up here in the latest Git diff.  We can't have non-const globals, and this extern indicates it is still here.